### ISSUE TITLE
Tracer is restarting auditd with the wrong command

### ISSUE THREAD

**Issue by paulrosberg** [FrostyX/tracer#192]
Using tracer that comes with katello-host-tools-tracer-3.5.7-5.el9.noarch.
So I guess the binaries:
tracer-common-0.7.8-1.el9.noarch
python3-tracer-0.7.8-1.el9.noarch

```
# tracer --version
0.7.8
```

Noticed tracer tries to restart auditd with the command “sudo systemctl restart auditd”.
However auditd should be restarted with “sudo service auditd restart” according to: https://access.redhat.com/solutions/2664811
Tested this on my EL 7/8/9 hosts and is correct. Can this be fixed?

**Comment by xsuchy** [comment]
Indeed. See /usr/libexec/initscripts/legacy-actions/auditd/restart

```
# sudo systemctl restart auditd
Failed to restart auditd.service: Operation refused, unit auditd.service may be requested by dependency only (it is configured to refuse manual start/stop).
See system logs and 'systemctl status auditd.service' for details.
```

**Comment by FrostyX** [comment]
Thank you for the feedback @paulrosberg,
I pushed a fix in 831fd91
